handlers.py

```diff
@@ -121,6 +121,37 @@ def handle_register_car(server, body, source):
     }
     _send_JSON(server, source, ackJSON)
 
+def handle_connect_car(server, body, source):
+    print('CONNECT CAR')
+
+    car_id = body['car_id']
+
+    if not car_id:
+        print('missing field: car_id')
+        server.send(Error.json(Error.BAD_REQ, 'missing field: car_id'), source)
+        return
+
+    dbconnect, cursor = _connect_to_db()
+    cursor.execute('select * from cars where (id=?)', (car_id,))
```

Comment:
Is the comma after `car_id` meant to be there?

Reply:
Yeah, it lets Python know that it's a tuple and not just an int.

Reply:
Dope.